homework1/task02.py

```diff
@@ -0,0 +1,30 @@
+"""
+Given a cell with "it's a fib sequence" from slideshow,
+    please write function "check_fib", which accepts a Sequence of integers,
+    and returns if the given sequence is a Fibonacci sequence
+
+We guarantee, that the given sequence contain >= 0 integers inside.
+
+"""
+from typing import Sequence
+
+
+def check_fibonacci(data: Sequence[int]) -> bool:
+
```

Comment:
your squence should be length of 3 to be the fibb

Reply:
Fixed